cranelift/codegen/src/prelude.isle

```diff
@@ -129,6 +129,9 @@
 (decl output_builder_finish (InstOutputBuilder) InstOutput)
 (extern constructor output_builder_finish output_builder_finish)
 
+(decl abi_arg_slot_regs (WritableValueRegs) ABIArg)
+(extern extractor abi_arg_slot_regs abi_arg_slot_regs)
```

Comment:
It doesn't look like this is actually invoked from ISLE -- I guess it's here so it can be in the `Context` trait and so the impl can sit next to its user in `isle.rs`?

I think I'd prefer to move it to a separate `impl IsleContext` block (ie not the trait impl) and avoid listing it here, just to avoid confusion ("why is this an extractor, where is it used?") if for no other reason...

Reply:
When my previous attempt had more of the lowering implemented in ISLE, I was using it there. I agree that it's odd to see it here, only to be used from the rust side -- we can always add it back to the prelude if it turns out it's necessary.